src/org/zaproxy/zap/extension/spider/SpiderAPI.java

```diff
@@ -138,6 +140,8 @@ public SpiderAPI(ExtensionSpider extension) {
 		this.addApiView(new ApiView(VIEW_FULL_RESULTS, new String[] { PARAM_SCAN_ID }));
 		this.addApiView(new ApiView(VIEW_SCANS));
 		this.addApiView(new ApiView(VIEW_EXCLUDED_FROM_SCAN));
+		this.addApiView(new ApiView(VIEW_URLS, new String[] { PARAM_SCAN_ID }));
```

Comment:
This view seems to duplicate `VIEW_RESULTS` (but depends on following comments).